packages/help-center/src/components/help-center-container.tsx

```diff
@@ -72,6 +73,8 @@ const HelpCenterContainer: React.FC< Container > = ( {
 		'is-minimized': isMinimized,
 	} );
 
+	useActionHooks();
```

Comment:
I simply moved this one level down because it now requires to be wrapped with React Router (it calls `useLocation`).